### ISSUE TITLE
The question of Pre-training

### ISSUE THREAD

**Issue by fishermanfff** [Vincent-ZHQ/UniS-MMC#4]
In your paper, the accuracy of the food data set reached 94%. What version of bert and vit did you use? When I reproduced your code, I found that you only provided the base version.

**Comment by lu-student** [comment]
Hello, dataloader.py part, where is the file in the /text_txt/ folder in this code text_path = self.args.data_dir + '/texts_txt/' + annotation + '/' + id.replace(".jpg", ".txt").

**Comment by lu-student** [comment]
![Uploading Snipaste_2024-01-08_09-52-10.png…]()

**Comment by Vincent-ZHQ** [comment]
> In your paper, the accuracy of the food data set reached 94%. What version of bert and vit did you use? When I reproduced your code, I found that you only provided the base version.

The results are all from models with BERT-base and ViT-base. I also test it with the large version. It could provide better results with larger GPU memory.
